casper/src/main/scala/coop/rchain/casper/rholang/RuntimeSyntax.scala

```diff
@@ -569,9 +570,9 @@ final class RuntimeOps[F[_]: Sync: Span: Log](
       )(validatorsPar => validatorsPar.size == 1)
       .map(validatorsPar => toValidatorSeq(validatorsPar.head))
 
-  def computeBonds(hash: StateHash): F[Seq[Bond]] =
+  def computeBonds(hash: StateHash, shardId: String): F[Seq[Bond]] =
     // Create a deploy with newly created private key
-    playExploratoryDeploy(bondsQuerySource, hash)
+    playExploratoryDeploy(bondsQuerySource, hash, shardId)
```

Comment:
When we are executing exploratory deploys and read only Rholang we don't really care about deploys, so you can just set shardId to empty string.

Reply:
> When we are executing exploratory deploys and read only Rholang we don't really care about deploys, so you can just set shardId to empty string.

Yes, I agree. I fixed it and related code in [commit](https://github.com/rchain/rchain/pull/3631/commits/6b3323cd4d0558d4385132b75632a09a1bcaad60).